model_training/script.sh

```diff
@@ -176,23 +182,35 @@ function collect() {
     collect_data stressng cpe-operator-system 60
 }
 
+function custom_collect() {
+    custom_collect_data customBenchmark
+}
+
 function quick_collect() {
     collect_data sample cpe-operator-system 10
 }
 
 function train() {
-    train_model stressng_kepler_query ${VERSION}
+    train_model stressng_kepler_query ${VERSION}_stressng
```

Comment:
I would like to keep only passing $VERSION here.
To separate custom pipeline, `PIPELINE_PREFIX` should be set.

Reply:
I have set this `suffix` similar to the way it is set for the sample benchmark [here](https://github.com/sustainable-computing-io/kepler-model-server/blob/main/model_training/script.sh#L188).
Thus, in the source code, if the benchmark == `sample`, then it is set to `_sample`, if the benchmark == `stressng`, then it is set to `_stressng`, and if the benchmark == `customBenchmark`, then it is set to `_customBenchmark`. Also, I assume that `PIPELINE_PREFIX` can be used to set the appropriate prefix (where as we are setting here a suffix).
I hope this addresses your comment.

Reply:
Got it. This also makes sense to me.